pulp_smash/tests/pulp3/file/api_v3/test_crud_importers.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
```

Comment:
@dkliban, I think this file itself should be renamed to `test_crud_remotes.py`. 
This will imply in update documentation as well.
See : https://github.com/PulpQE/pulp-smash/blob/master/docs/api.rst